main.nf

```diff
@@ -22,37 +23,47 @@ version = '0.1'
 // Configurable variables - same as NGI-RNAseq for now
 params.project = false
 params.reads = "data/*{1,2}.fastq.gz"
-params.outdir = './results'
 params.email = false
-params.star = false
+params.star = true
+params.inspector = false
 params.fusioncatcher = true
+params.sensitivity = 'sensitive'
+params.clusterOptions = false
+params.outdir = './results'
+params.fc_extra_options = ''
 Channel
     .fromFilePairs( params.reads, size: 2 )
     .ifEmpty { exit 1, "Cannot find any reads matching: ${params.reads}" }
     .into { read_files_star_fusion; fusion_inspector_reads; fusioncatcher_reads}
 
 
 /*
- * STEP 1 - STAR-Fusion
+ * STAR-Fusion
  */
 process star_fusion{
-
+    tag "$name"
+    publishDir "${params.outdir}/Star_fusion",  mode: 'copy'
+    
     input:
-    set val (name), file(read1), file(read2) from read_files_star_fusion
+    set val (name), file(reads) from read_files_star_fusion
 
     output:
     file '*final.abridged*' into star_fusion_abridged
-    file 'star-fusion.fusion_candidates.final.abridged.FFPM' into fusion_candidates
+    file '*star-fusion.fusion_candidates.final.abridged.FFPM' into fusion_candidates,fusion_candidates_list
 
     when: params.star
 
     script:
     """
-    STAR-Fusion \
-        --genome_lib_dir $star_fusion_refrence \
-        --left_fq $read1 \
-        --right_fq $read2 \
-        --output_dir $name
+    STAR-Fusion \\
+        --genome_lib_dir ${params.star_fusion_reference}\\
```

Comment:
I imagine that you'll have to stage this as an input channel to get the docker / singularity mounts to work..

eg:
https://github.com/SciLifeLab/NGI-RNAseq/blob/master/main.nf#L163-L165
https://github.com/SciLifeLab/NGI-RNAseq/blob/master/main.nf#L587